<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabelMouseTracking" name="lblTopRight">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 069, 019, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabelMouseTracking" name="lblTopLeft">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 069, 019, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabelMouseTracking" name="lblTop">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 169, 119, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabelMouseTracking" name="lblLeft">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 169, 119, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabelMouseTracking" name="lblRight">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 169, 119, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabelMouseTracking" name="lblBottomLeft">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 069, 019, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabelMouseTracking" name="lblBottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 169, 119, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabelMouseTracking" name="lblBottomRight">
      <property name="maximumSize">
       <size>
        <width>6</width>
        <height>6</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(139, 069, 019, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabelMouseTracking" name="lblCenter">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 235, 205, 0.7)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLabelMouseTracking</class>
   <extends>QLabel</extends>
   <header location="global">qlabelmousetracking.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
